app/webpacker/styles/font-sizes.scss

```diff
@@ -1,29 +1,56 @@
 html {
-  font-size: 100%;
+  @include mq($until: mobile) { font-size: 85%; }
+  @include mq($from: mobile, $until: tablet) { font-size: 92%; }
+  @include mq($from: tablet) { font-size: 100%; }
 }
 
 // Numbers refer to pixel size when the base font
 // size of the broswer is 16px (most broswers)
 
+$fs-12: .8rem;
+$fs-14: .9rem;
 $fs-16: 1rem;
 $fs-19: 1.1875rem;
 $fs-28: 1.75rem;
 $fs-36: 2.25rem;
 $fs-42: 2.625rem;
 $fs-54: 3.375rem;
 
-$sizes: (
-  "xsmall": $fs-16,
-  "small": $fs-19,
-  "medium": $fs-28,
-  "large": $fs-36,
-  "xlarge" : $fs-42,
-  "xxlarge": $fs-54
-);
-
-@function size($size) {
-  @if not map-has-key($sizes, $size) {
-    @error "Unknown font size `#{$size}`";
+@function size($device-sizes, $size) {
+  @if not map-has-key($device-sizes, $size) {
+    @error "Unknown font size `#{$size}` in #{$device-sizes}";
   }
-  @return map-get($sizes, $size);
+  @return map-get($device-sizes, $size);
+}
+
+$mobile-sizes:  ("xsmall": $fs-12, "small": $fs-14, "medium": $fs-16, "large": $fs-19, "xlarge" : $fs-28, "xxlarge": $fs-36);
+$tablet-sizes:  ("xsmall": $fs-14, "small": $fs-16, "medium": $fs-19, "large": $fs-28, "xlarge" : $fs-36, "xxlarge": $fs-42);
+$desktop-sizes: ("xsmall": $fs-16, "small": $fs-19, "medium": $fs-28, "large": $fs-36, "xlarge" : $fs-42, "xxlarge": $fs-54);
+
+@mixin font-size($size) {
+  @include mq($until: mobile) {
+    font-size: size($mobile-sizes, $size);
+  }
+
+  @include mq($from: mobile, $until: tablet) {
+    font-size: size($tablet-sizes, $size);
+  }
+
+  @include mq($from: tablet) {
+    font-size: size($desktop-sizes, $size);
+  }
+
+  // debug util
+  //
+  // $debug-colours: (
+  //   "xsmall": powderblue,
+  //   "small": plum,
+  //   "medium": coral,
+  //   "large": lawngreen,
+  //   "xlarge" : fuchsia,
+  //   "xxlarge": mediumspringgreen);
+
+  // @each $name, $whatever in $desktop-sizes {
+  //   outline: 1px dashed map-get($debug-colours, $size);
+  // }
```

Comment:
Left this in because it's super-useful to future people working out wtf is going on with our font sizes 😅

Reply:
v nice debugging!